realm/src/main/java/io/realm/Realm.java

```diff
@@ -1413,12 +1412,12 @@ public static void migrateRealmAtPath(String realmPath, byte[] key, RealmMigrati
         migrateRealmAtPath(realmPath, key, migration, true);
     }
 
-    @SuppressWarnings("UnusedDeclaration")
     public static void migrateRealmAtPath(String realmPath, RealmMigration migration, boolean autoRefresh) {
         migrateRealmAtPath(realmPath, null, migration, autoRefresh);
     }
 
-    public static void migrateRealmAtPath(String realmPath, byte[] key, RealmMigration migration, boolean autoUpdate) {
+    public static synchronized void migrateRealmAtPath(String realmPath, byte[] key, RealmMigration migration,
+                                            boolean autoUpdate) {
```

Comment:
Yes. Max 120 chars lines. That's the convention I thought we were using. I set up Studio to split on that.